src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -3,7 +3,52 @@
 <head>
     <meta charset="utf-8">
     <title>{0}</title>
+    <script>  
+      (function() {{
+          const logPrefix = '[Starcounter App Shell]';
+          function register() {{
+          if ('serviceWorker' in navigator) {{
```

Comment:
Can you add a `console.info` for browsers that do fail the check `'serviceWorker' in navigator`?

This happens for example when trying to run http://backoffice.omni.local:8080/BlendingEditor in Omni (because it is neither HTTPS nor localhost)